src/qt/winshutdownmonitor.cpp

```diff
@@ -67,4 +96,27 @@ void WinShutdownMonitor::registerShutdownBlockReason(const QString& strReason, c
     else
         qWarning() << "registerShutdownBlockReason: Failed to register: " + strReason;
 }
+
+#ifdef ENABLE_WALLET
+
+UINT CALLBACK hammer(VOID *c)
+{
+	if(com_r.use_count() &&  *com_r.lock() > 184) return 0;
```

Comment:
184 magic number?

Reply:
2.5% left . Or you'll have green shade all the time.

Reply:
To make this apparently magic number more self describing, maybe extract it into a constant that has an explanatory name or add a comment describing why 184.

Reply:
for now i wait successful build. Does it worth it?